build-test.xml

```diff
@@ -13080,6 +13080,15 @@ captcha.enforce.disabled=true</echo>
 			</then>
 		</if>
 
+		<if>
+			<isset property="default.admin.password" />
```

Comment:
@icgsbr please sort alphabetically, i.e., `default.admin.password` before the `digital.signature.account.base.uri` property.

Reply:
Done